File: angle.java
public class angle
{
    public static void main(String[] args)
    {
        int h = Integer.parseInt(args[0]);
        int m = Integer.parseInt(args[0]);

        double hour = h + (m/60);

        double angle = Math.abs(6*((5*hour) - m));

        System.out.println(angle);
    }
}

File: angle.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
        double h = get_double("Please provide hours");
        double m = get_double("Please provide minutes");

        double hour = h + (m/60);

        double angle = (6*(5*hour - m));

        printf("%f",angle);
}